src/math/Color.js

```diff
@@ -56,6 +56,10 @@ class Color {
 
 	constructor( r, g, b ) {
 
+		this.r = 1;
```

Comment:
Maybe it would be better to add these three lines as a fallback in `Color.set()`? Then fhey can be removed from here.

Reply:
TBH, I think defining the properties in the ctor is more readable. And the place where you would expect the property definition.